src/test/regress/columnar_am_schedule

```diff
@@ -0,0 +1,18 @@
+test: am_create
```

Comment:
is it possible to run the tests in parallel?

Reply:
Some depend on the artifacts left by others. We can probably find some that can run in parallel.
However, the runtime is 33 seconds :) compared to other suites in our test it is neglectable :)